eventfiles/tests.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+from .models import EventFile
+from django.core.files import File
+from django.core.exceptions import ValidationError
+from events.tests import event_participant_not_creator, new_event, user0  # noqa: F401
+
+
+TEST_FILE_PATH = 'static/test_files/testFile.txt'
+FILE_ALREADY_EXISTS = 'that file already exist in meeting'
+
+
+@pytest.fixture()
+def file0():
+    return File(open(TEST_FILE_PATH))
+
+
+@pytest.fixture()
+def event_file0(file0, event_participant_not_creator):  # noqa: F811
+    return EventFile(file=file0, participant_id=event_participant_not_creator)
+
+
+@pytest.fixture()
+def persist_event_file(event_file0):
+    event_file0.participant_id.event_id.save()
+    event_file0.participant_id.user_id.save()
+    event_file0.participant_id.save()
+    event_file0.save()
+    return event_file0
+
+
+@pytest.mark.django_db()
```

Comment:
maybe instead of repeating the django_db a few times you can create a class with it for all tests like:
```suggestion
@pytest.mark.django_db
class TestEventFile:
```